lib/Command/CirclesMemberships.php

```diff
@@ -204,7 +204,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		if ($count === 0) {
 			$output->writeln('(database not updated)');
 		} else {
-			$output->writeln('(' . $count . ' entries generated/updated in the database)');
+			$output->writeln('(' . ((string)$count) . ' entries generated/updated in the database)');
```

Comment:
Why are all these casts needed, it’s perfectly fine to concatenate an int with a string, no?